sound/soc/sof/topology.c

```diff
@@ -2508,10 +2508,13 @@ int snd_sof_load_topology(struct snd_soc_component *scomp, const char *file)
 
 	if (!sof_pdata->disable_function_topology && !disable_function_topology &&
 	    sof_pdata->machine && sof_pdata->machine->get_function_tplg_files) {
+		bool no_fallback = strstr(file, "dummy");
```

Comment:
This bit would benefit from some hint (in code comment?) why "dummy" should change the behaviour. I think it's ok to have the longer story in commit message, but something in code would be helpful why this check is done.